fs/rpmsgfs/rpmsgfs_server.c

```diff
@@ -285,6 +285,9 @@ static int rpmsgfs_attach_dir(FAR struct rpmsgfs_server_s *priv,
       return -ENOMEM;
     }
 
+  memset(&tmp[priv->dir_nums], 0, sizeof(FAR void *) *
```

Comment:
I think & is not required here because the variable is **tmp

Reply:
ping @acassis

Reply:
Ok, gotcha!